buildbot-config/platforms.py

```diff
@@ -261,15 +261,26 @@ def appletv():
 appletv()
 
 def atari():
-    platform = Platform("atari")
+    platform = Platform("atari-full")
+    platform.workerimage = "atari"
     platform.compatibleBuilds = (builds.ScummVMBuild, )
 
-    platform.env["ASFLAGS"] = "-m68020-60"
-    platform.env["CXXFLAGS"] = "-m68020-60 -DUSE_MOVE16 -DUSE_SUPERVIDEL -DUSE_SV_BLITTER -DDISABLE_LAUNCHERDISPLAY_GRID"
-    platform.env["LDFLAGS"] = "-m68020-60"
+    additional_flags = {
+        'ASFLAGS': '',
+        'CFLAGS': '',
+        'CPPFLAGS': '',
+        'CXXFLAGS': '-DUSE_MOVE16 -DUSE_SUPERVIDEL -DUSE_SV_BLITTER -DDISABLE_LAUNCHERDISPLAY_GRID',
+        'LDFLAGS': '-L${PREFIX}/lib/m68020-60',
```

Comment:
Why `-L` has to be listed here explicitly? Is there a reason why it can't be included in the default `LDFLAGS`?